Issues on launching docs dev mode

### Environment

On StackBlitz
------------------------------
- Operating System: Linux
- Node Version:     v16.20.0
- Nuxt Version:     3.7.3
- CLI Version:      3.8.4
- Nitro Version:    2.6.3
- Package Manager:  pnpm@8.6.10
- Builder:          -
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### Reproduction

https://stackblitz.com/~/github.com/nuxt/nuxt

Stop when starting the playground.

Then, start `pnpm docs:dev`.

### Describe the bug

`Cannot find module '@nuxt/kit'` when starting the dev mode.

Actually, it's resolvable by installing deps in `.website` folder and running manually the dev script from the folder. Then, when the first start is done, redoing the start from root is working.

### Additional context

It seems that is caused by a combination of nuxi issue and the fact .website deps are not installed.

### Logs

```shell-script
ERROR  Cannot find module '@nuxt/kit'                                                                                 00:26:25
Require stack:
- /home/nuxt/nuxt/node_modules/.pnpm/@nuxt-themes+docus@1.14.6_nuxt@packages+nuxt_postcss@8.4.29_vue@3.3.4/node_modules/@nuxt-themes/docus/nuxt.config.ts

  Require stack:
  - node_modules/.pnpm/@nuxt-themes+docus@1.14.6_nuxt@packages+nuxt_postcss@8.4.29_vue@3.3.4/node_modules/@nuxt-themes/docus/nuxt.config.ts
  at InternalError (https://nuxt-eboq.w-corp.staticblitz.com/blitz.ef0c375a.js:67:6226)
  at Module._resolveFilename (https://nuxt-eboq.w-corp.staticblitz.com/blitz.ef0c375a.js:57:12070)
  at resolve (https://nuxt-eboq.w-corp.staticblitz.com/blitz.ef0c375a.js:91:2142)
  at _resolve (node_modules/.pnpm/jiti@1.20.0/node_modules/jiti/dist/jiti.js:1:250403)
  at jiti (node_modules/.pnpm/jiti@1.20.0/node_modules/jiti/dist/jiti.js:1:252994)
  at  (node_modules/.pnpm/@nuxt-themes+docus@1.14.6_nuxt@packages+nuxt_postcss@8.4.29_vue@3.3.4/node_modules/@nuxt-themes/docus/nuxt.config.ts:1:181)
  at evalModule (node_modules/.pnpm/jiti@1.20.0/node_modules/jiti/dist/jiti.js:1:255691)
  at jiti (node_modules/.pnpm/jiti@1.20.0/node_modules/jiti/dist/jiti.js:1:253611)
  at resolveConfig (node_modules/.pnpm/c12@1.4.2/node_modules/c12/dist/index.cjs:300:24)
  at extendConfig (node_modules/.pnpm/c12@1.4.2/node_modules/c12/dist/index.cjs:237:27)
  at loadConfig (node_modules/.pnpm/c12@1.4.2/node_modules/c12/dist/index.cjs:179:11) 



 ERROR  Cannot find module '@nuxt/kit'                                                                                 00:26:25
Require stack:
- /home/nuxt/nuxt/node_modules/.pnpm/@nuxt-themes+docus@1.14.6_nuxt@packages+nuxt_postcss@8.4.29_vue@3.3.4/node_modules/@nuxt-themes/docus/nuxt.config.ts
```


Discord thread: https://discord.com/channels/473401852243869706/1151233212849336410

--- a/docs/5.community/5.framework-contribution.md
+++ b/docs/5.community/5.framework-contribution.md
@@ -128,7 +128,7 @@ git checkout -b my-new-branch
 
 ### Set Up Documentation Website in Local Environment
 
-The Nuxt documentation is currently deployed within [nuxt/nuxt in `docs` folder](https://github.com/nuxt/nuxt/tree/main/docs).
+The Nuxt documentation is currently deployed within [nuxt/nuxt.com](https://github.com/nuxt/nuxt.com) as a layer.
 
 - Run `pnpm build:stub` once in the root directory
 - Go into the `.website` directory: `cd .website`

--- a/docs/5.community/5.framework-contribution.md
+++ b/docs/5.community/5.framework-contribution.md
@@ -128,12 +128,13 @@ git checkout -b my-new-branch
 
 ### Set Up Documentation Website in Local Environment
 
-The Nuxt documentation is currently deployed within [nuxt/nuxt.com](https://github.com/nuxt/nuxt.com) as a layer.
+The Nuxt documentation is currently deployed within [nuxt/nuxt in `docs` folder](https://github.com/nuxt/nuxt/tree/main/docs).
 
 - Run `pnpm build:stub` once in the root directory
-<!-- - Go into the docs directory: `cd docs` -->
-<!-- - Install docs dependencies using `yarn install` -->
-<!-- - Run `yarn dev` to start docs in development mode -->
+- Go into the `.website` directory: `cd .website`
+- Install dependencies using `pnpm install`
+<!-- TODO: make 2 previous steps transparent by running pnpm docs:dev from root directory -->
+- Run `pnpm dev` to start docs in development mode
 - Before opening a PR, run `pnpm lint:docs:fix` to highlight and resolve any lint issues
 
 ::alert

